Formula/commandbox.rb

```diff
@@ -11,15 +11,15 @@ class Commandbox < Formula
   end
 
   bottle do
-    sha256 cellar: :any_skip_relocation, all: "16a80454d87d885fd04543ccf315ac60526d694909a9bc235017f510bcd74008"
+    sha256 cellar: :any_skip_relocation, all: "45b01d0263bf8842d6b060ed23202a77ab6b33d2a3c1c2a89be80eb4c50324ba"
```

Comment:
Please don't touch the `bottle` block.

Reply:
Sorry, the bottle is automatically updated by some process after every pull so I didn't think it mattered.  That's just whatever values was in my template I use.

Reply:
@carlocab I have put the bottle block back.  Again, not sure why it maters since it will be overwritten by one of your automated processes as soon as it's merged, but there you go :)

Reply:
It matters because it muddies the Git log, which `brew` parses -- we shouldn't do anything that might break that. Also, it's not a good security practise to commit a checksum for a bottle that @BrewTestBot never created into this repo.

Reply:
@carlocab Actually, brew test bot DID create that checksum. That was the first value that was auto-committed back when you guys added that feature.  I copied it into my template file that I update in my build that gets used for my tap pin plus I use it to copy paste here every time I need to create a pull. The second pull I sent after the bottle was introduced I noticed the value had changed so I asked what I was supposed to do.  No one ever explained what the heck the value even is (and I still don't know to this day) but they basically told me not to worry about it because a bot would come along and change it anyway.  And I've never worried about it since. Frankly, I wish you would remove it as it's a pain in the butt to automate sending my pulls a much as possible when there are things that change in the file every time.  I would really like to just be able to copy over a template file for the pull and be done.  FWIW, I don't use Brew, or Macs, nor do I code in Ruby.  I only maintain this formula for my Mac users, so this entire pull request process is the only manual thing I have to do to release and it's pretty annoying.